drivers/sx127x/sx127x.c

```diff
@@ -42,20 +42,20 @@
 #include "debug.h"
 
 /* The reset signal must be applied for at least 100 µs to trigger the manual
-   reset of the device. To ensure this value is big enough even with an
-   inaccurate clock source, an additional 10 % error margin is added. */
-#define SX127X_MANUAL_RESET_SIGNAL_LEN_US       (110U)
+   reset of the device. In order to avoid a dependency to the high frequency
+   timers, we round it to 1 ms */
+#define SX127X_MANUAL_RESET_SIGNAL_LEN_MS       (1U)
 
 /* After triggering a manual reset the device needs at least 5 ms to become
-   ready before interacting with it. To ensure this value is big enough even
-   with an inaccurate clock source, an additional 10 % error margin is added. */
-#define SX127X_MANUAL_RESET_WAIT_FOR_READY_US   (5500U)
+   ready before interacting with it. We round up to 6 ms in case the clock
+   source is not accurate enough */
+#define SX127X_MANUAL_RESET_WAIT_FOR_READY_MS   (6U)
```

Comment:
I think the rounding up would not be needed, ztimer guarantees *at least* semantics. Not waiting long enough would be a ztimer bug. But I guess that 1ms extra wait doesn't hurt either...

Reply:
Yeah, but I can imagine is the clock source is drifting forward the actual timer fires before the expected value, even though the semantics of ztimer guarantees "at least" semantics. Or I'm missing something?

Reply:
or did you mean the clock source of the sx127x? then it makes sense of course

Reply:
yes, I meant the clock source :). Ok, so I guess we can leave it as it is.